### ISSUE TITLE
backend/caldav.php, line 109, $account['url'] is not defined

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#322]
```
File calendar/program/backend/caldav.php, revision 81, in connect(), line 109

Any step that executes connect() will have a PHP warning because 
$account['url'] is not defined, it should be just $url.

From:
write_log('calendar',"CalDAV Invalid URL: $user - '".$account['url']."'");

To:
write_log('calendar',"CalDAV Invalid URL: $user - '$url'");

And i would like to suggest to add an if(something) before that write_log(), 
because some users may not have $user defined and it will be writing 
unnecessary logs (if the user didn't define a username, it's not really an 
invalid URL).
```

Original issue reported on code.google.com by `diana.so...@gmail.com` on 2 Apr 2012 at 6:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 3 Apr 2012 at 4:39
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
OK?

{{{
      if(!$ret){
        if(!$user)
          write_log('calendar',"CalDAV Invalid URL: $user - $url");
        $ret = $this->caldav = new CalDAVClient('', '', '', '');
      }
}}}
```

Original comment by `rol...@liebl.ath.cx` on 3 Apr 2012 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
Here i did the same.
Thank you for this project.
```

Original comment by `diana.so...@gmail.com` on 3 Apr 2012 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
Fix in calendar v8.2.
```

Original comment by `rol...@liebl.ath.cx` on 5 Apr 2012 at 4:34
- Changed state: **Fixed**
